src/sentry/interfaces/stacktrace.py

```diff
@@ -664,26 +664,17 @@ def to_python(cls, data, has_system_frames=None, slim_frames=True,
         else:
             kwargs['frames_omitted'] = None
 
-        kwargs['has_system_frames'] = has_system_frames
-
         instance = cls(**kwargs)
         if slim_frames:
             slim_frame_data(instance)
         return instance
 
-    @classmethod
-    def data_has_system_frames(cls, data):
-        system_frames = 0
-        for frame in data['frames']:
-            # XXX(dcramer): handle PHP sending an empty array for a frame
-            if not isinstance(frame, dict):
-                continue
-            if not frame.get('in_app'):
-                system_frames += 1
-
-        if len(data['frames']) == system_frames:
-            return False
-        return bool(system_frames)
+    def get_has_system_frames(self):
```

Comment:
This is called `get_has_system_frames` to not be named` has_system_frames` which for old data overrides the class level method with a attribute from the data.